### PR TITLE
[DOCS-2554] add sigv4

### PR DESCRIPTION
**Pull request by HugoPct**
### Preview

https://docs-staging.datadoghq.com/hugo.puceat/sigv4/synthetics/multistep
https://docs-staging.datadoghq.com/hugo.puceat/sigv4/synthetics/api_tests/http_tests/

### Additional Notes
Open Question is it possible to embed the table into the 2. ?
I found it a bit weird to hop from the tab content page to the main page.
https://a.cl.ly/jku4GNXv
---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/api_tests/http_tests.md [hunk 1]
```diff
@@ -43,7 +43,13 @@ After choosing to create an [`HTTP` test][4], define your test's request.
   * **Follow redirects**: Select to have your HTTP test follow up to ten redirects when performing the request.
   * **Request headers**: Define headers to add to your HTTP request. You can also override the default headers (for example, the `user-agent` header).
   * **Cookies**: Define cookies to add to your HTTP request. Set multiple cookies using the format `<COOKIE_NAME1>=<COOKIE_VALUE1>; <COOKIE_NAME2>=<COOKIE_VALUE2>`.
+
+  {{% /tab %}}
+
+  {{% tab "Authentication" %}}
+
   * **HTTP Basic Auth**: Add HTTP basic authentication credentials.
+  * **AWS Signature V4**: Enter your Access Key ID and Secret Access Key. Datadog will generate the signature for your request.
```
Comment (kayayarai) [inline content/en/synthetics/api_tests/http_tests.md:52]:
```suggestion
  * **AWS Signature V4**: Enter your Access Key ID and Secret Access Key. Datadog generates the signature for your request.
```

File: content/en/synthetics/multistep.md [hunk 2]
```diff
@@ -67,7 +67,13 @@ To create an HTTP request step, click **Create Your First Step**.
   * **Follow redirects**: Tick to have your HTTP test follow up to ten redirects when performing the request.
   * **Request headers**: Define headers to add to your HTTP request. You can also override the default headers (for example, the `user-agent` header).
   * **Cookies**: Define cookies to add to your HTTP request. Set multiple cookies using the format `<COOKIE_NAME1>=<COOKIE_VALUE1>; <COOKIE_NAME2>=<COOKIE_VALUE2>`.
+
+  {{% /tab %}}
+
+  {{% tab "Authentication" %}}
+
   * **HTTP Basic Auth**: Add HTTP basic authentication credentials.
+  * **AWS Signature V4**: Enter your Access Key ID and Secret Access Key. Datadog will generate the signature for your request.
```
Comment (kayayarai) [inline content/en/synthetics/multistep.md:76]:
```suggestion
  * **AWS Signature V4**: Enter your Access Key ID and Secret Access Key. Datadog generates the signature for your request.
```

**Review by kayayarai — APPROVED** [review]

**Comment by kayayarai** [comment]
Checking into your question before I merge

**Comment by kayayarai** [comment]
And the answer is Yes! Aligned the tabs shortcodes with the list indentation (three spaces). Will let the preview site build so you can see.

**Comment by HugoPct** [comment]
Wouah awesome!
thank you Kari

**Pull request merged**